src/views/Dashboard/Customer/SelectLab/index.vue

```diff
@@ -0,0 +1,243 @@
+import AgreementDialog from '@/common/components/Dialog/AgreementDialog.vue'
+
+
+export default {
+  name: "SelectLab",
+    components: {
+    Stepper,
+    MenuCard,
+    Button,
+    Dialog,
+    AgreementDialog,
+  },
+
+  data: () => ({
+    stepperItems: [
+      { number: 1, title: 'Select a location and Service Category', active: false },
+      { number: 2, title: 'Select a lab', active: true },
+      { number: 3, title: 'Checkout', active: false },
+      { number: 4, title: 'Payment', active: false },
+    ],
+    showNoLab: false,
+    showAgreement: false,
+    showSuccessDialog: false
+  }),
+
+  computed: {
+    ...mapState({
+      country: (state) => state.lab.country,
+      city: (state) => state.lab.city,
+      category: (state) => state.lab.category,
+      labs: (state) => state.lab.labs
+    }),
+  },
+
+  async mounted () {
+    if (!this.labs.length) {
+      this.showNoLab = true
+    }
+  },
+
+  methods : {
+
+    ...mapMutations({
+      setLabToRequest: "testRequest/SET_LAB",
+      setProductsToRequest: "testRequest/SET_PRODUCTS",
+    }),
+
+    getServiceName(services) {
+      for (let i = 0; i < services.length; i++) {
+        if (services[i].info.category == this.category) {
+          return services[i].info.name
+        }
+      }
+    }, 
+
+    getServiceDescription(services) {
+      for (let i = 0; i < services.length; i++) {
+        if (services[i].info.category == this.category) {
+          return services[i].info.description
+        }
+      }
+    },
+
+    getServicePrice(services) {
+      for (let i = 0; i < services.length; i++) {
+        if (services[i].info.category == this.category) {
+          return services[i].info.prices_by_currency[0].total_price
+        }
+      }
+    },
+
+    getServiceImage(services) {
+      for (let i = 0; i < services.length; i++) {
+        if (services[i].info.category == this.category) {
+          return services[i].info.image
+        }
+      }
+    },
+    async selectLab(lab, services) {
+
+      let selectedProduct
+      for (let i = 0; i < services.length; i++) {
+        if (services[i].info.category == this.category) {
+          selectedProduct = services[i]
+        }
+      }
+      
+      this.setProductsToRequest(selectedProduct)
+      this.setLabToRequest(lab)
+      this.$router.push({ name: 'request-test-checkout' })
+    },
+
+    showingAgreementDialog () {
+      console.log("showingg...")
+      this.showAgreement = true
+    },
+
+    closingDialog() {
+      this.showAgreement = false
+    },
+
+    onClickDialogButton() {
+      console.log("kesini ?")
+      this.showAgreement = false
+      this.showSuccessDialog = true
+    },
+
+    toHomePage () {
+      this.$router.push({ name: 'customer-dashboard' })
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.select-lab {
+    width: auto;
+    height: 732px;
+    background: #FFFFFF;
+}
+
+.title {
+  font-family: 'Raleway';
+  margin-top: 5%;
```

Comment:
as i said before you will repeat your code,, i think u need to use our sass mixins